### ISSUE TITLE
TS 4.5.0-dev.20211029: JSX attrbutes not showing in intellisense 

### ISSUE THREAD

**Issue by markusjohnsson** [microsoft/vscode#136377]
# Bug Report

### 🔎 Search Terms

jsx tsx props attributes intellisense not working after newline

### 🕗 Version & Regression Information

- This changed between versions 4.4.4 and 4.5.0-dev.20211029

### ⏯ Playground Link

Sorry, cannot use JSX in playground.

### 💻 Code

```ts

function MyComponent({ a, b }: { a: number; b: string; }) {
    return <div/>;
}

function Foo() {
    return <div>
        <MyComponent
                  (invoke intellisense here)
        />
    </div>
}
```

### 🙁 Actual behavior

In 4.5.0-dev.20211029, JSX/TSX props/attributes are not displayed in intellisense completion after a newline.

same line works:
![image](https://user-images.githubusercontent.com/557685/139419362-69b791eb-cd26-4446-a643-f64fac954a72.png)

after newline, does not work:
![image](https://user-images.githubusercontent.com/557685/139419421-8b11ac87-c102-4d3a-806a-f7875465f6e4.png)

### 🙂 Expected behavior

JSX/TSX props/attributes

4.4.4:
both works:
![image](https://user-images.githubusercontent.com/557685/139419520-e93959bc-83a2-41e3-895a-b2de76a24689.png)

![image](https://user-images.githubusercontent.com/557685/139419489-621b4cba-03a5-4ade-bb09-554828e4eba1.png)

**Comment by andrewbranch** [comment]
@markusjohnsson thanks for the reminder ☝️

**Comment by mjbvz** [comment]
@andrewbranch I believe the root cause is that ts now returns a `replacementSpan` in this example:

```ts
  "insertText": "b=\"$1\"",
            "replacementSpan": {
                "start": {
                    "line": 9,
                    "offset": 9
                },
                "end": {
                    "line": 9,
                    "offset": 10
                }
            },
```

Do you know why this span is being returned? It seems to map to the closing `/` in `/>` which seems incorrect

We probably need to fix this for November VS Code release (start of December)

**Comment by andrewbranch** [comment]
@mjbvz the fix for this is in the 4.5 branch now 👍

**Comment by mjbvz** [comment]
Thanks @andrewbranch! Closing as fixed upstream
